src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -0,0 +1,41 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class FigureSupplier {
+    private static final String DEF_FIGURE_COLOR = Color.WHITE.name().toLowerCase();
```

Comment:
Using 'toLowerCase()' may not be necessary for 'DEF_FIGURE_COLOR' if it is only used internally and not displayed to the user. If you do need to present it in lowercase, consider converting it at the point of output.